### DISCUSSION TITLE
SecurityContext propagation

### DISCUSSION THREAD

**Discussion by lucimihai** [graphql-java-kickstart/graphql-spring-boot#715] [Q&A]
I saw that release 12.0.0 would solve this issue.

When will it be released?

BR

**Comment by oliemansm** [comment]
It's released today

**Reply by krdk01** [reply]
When will The release 12.0.0  add to mvn center?

**Reply by oliemansm** [reply]
It was added to maven central yesterday: https://search.maven.org/artifact/com.graphql-java-kickstart/graphql-spring-boot-starter/12.0.0/jar
